test/attachment-constraints.spec.js

```diff
@@ -77,21 +76,10 @@ describe('File attachment constraints:', () => {
             attachmentRefProp: 'bar.html' // The attachmentReference's maximum size of 40 overrides the document's maximum individual size of 25
           };
 
-          let syncFuncError = null;
-          expect(() => {
-            try {
-              testFixture.testEnvironment.syncFunction(doc);
-            } catch (ex) {
-              syncFuncError = ex;
-
-              throw ex;
-            }
-          }).to.throw();
-
-          testFixture.verifyValidationErrors(
+          testFixture.verifyDocumentNotCreated(
```

Comment:
As mentioned in the pull request description, changes like this one are not directly related to the original purpose of issue #259, but they do eliminate unnecessary duplication of code already provided by the test fixture object.